tests/userSocialMedia.test.ts

```diff
@@ -0,0 +1,116 @@
+import faker = require('faker');
+import { SocialMediaType } from '../types';
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, PortalState, UserFactory } from './data';
+import { UserSocialMediaFactory } from './data/UserSocialMediaFactory';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('social media URL submission', () => {
+  test('properly persists on successful submission', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+    const userSocialMedia = UserSocialMediaFactory.fake();
+
+    await new PortalState()
+      .createUsers(member)
+      .write();
+
+    const userController = ControllerFactory.user(conn);
+
+    await userController.insertSocialMediaForUser({ socialMedia: userSocialMedia }, member);
+    const userResponse = await userController.getUser({ uuid: member.uuid }, member);
+
```

Comment:
In this block, to limit the number of controller calls to 1, let's use a `conn.manager.findOne()` call instead of the `getUser` call here, and maybe have 1 small unit test where we just check the `getUser()` route functionality. This would be the first time in the whole codebase that we're doing tests for specifically GET routes, but it's a convention we should set for future precedence to keep the 1-route-per-test standard.